### ISSUE TITLE
Package wishlist

### ISSUE THREAD

**Issue by IanButterworth** [ianshmean/YOLO.jl#3]
Models
- [ ] YOLO v2
- [x] YOLO v2-tiny
- [ ] YOLO v3
- [ ] YOLO v3-tiny
- [ ] custom model

Datasets
- [x] Optional downloading of datasets
- [x] VOC
- [ ] COCO
- [ ] ImageNet
- [ ] Custom datasets & dataset locations (possible but untested)

Testing
- [ ] Elegant minibatching
- [x] Move away from ImageView given Gtk issues on MacOS, to Makie

Training
- [ ] Loss function etc.

Backends
- [ ] Switchable backends
- [x] Knet
- [ ] Flux

**Comment by stemann** [comment]
Looking forward to see this realized. What can I do to help?

**Comment by IanButterworth** [comment]
@stemann I just merged a PR that makes this functional! Can you test, using the example code in the README? Much appreciated!!

**Comment by iuliancioarca** [comment]
Hi,
I managed to write the YOLOv2 chain and to load the pretrained weights (manually, no automatic generation). Forked repo here: https://github.com/iuliancioarca/YOLO.jl
I also updated the Readme with an example on how to load and use YOLOv2. Posprocessing and visualization were simply plug and play from V2_tiny.
